cli/test/org/partiql/cli/CliTest.kt

```diff
@@ -155,6 +155,30 @@ class CliTest {
         assertEquals("{a:1}\n{b:1}\n", actual)
     }
 
+    @Test
+    fun withSelectIn() {
+        makeCli(
+            query = "SELECT age FROM << { 'name': 'John', 'age': 22 } >> WHERE name IN (SELECT name FROM <<{ 'name': 'John' }>>)",
+            output = FileOutputStream(testFile),
+            outputFormat = OutputFormat.ION_TEXT
+        ).run()
+        val actual = testFile.bufferedReader().use { it.readText() }
+
+        assertEquals("{age:22}\n", actual)
+    }
+
+    @Test
+    fun withSelectInButEmpty() {
+        makeCli(
+            query = "SELECT age FROM << { 'name': 'John', 'age': 22 } >> WHERE name IN (SELECT name FROM <<{ 'name': 'john' }>>)",
+            output = FileOutputStream(testFile),
+            outputFormat = OutputFormat.ION_TEXT
+        ).run()
+        val actual = testFile.bufferedReader().use { it.readText() }
+
+        assertEquals("\n", actual)
+    }
```

Comment:
Anyway, looking again at your other tests, I don't think there's a need to test the CLI separately from the `EvaluatingCompilerInTests`.

Reply:
ACK. I can remove. Thanks!